bcda/models/models_test.go

```diff
@@ -79,6 +81,38 @@ func (s *ModelsTestSuite) TestCreateACO() {
 	assert.NotNil(err)
 }
 
+func (s *ModelsTestSuite) TestACOPublicKeySave() {
+	assert := s.Assert()
+
+	// Setup ACO
+	cmsID := "A9994"
+	var aco ACO
+	s.db.Find(&aco, "aco_id = ?", cmsID)
```

Comment:
There is no column "aco_id" in the acos table. This find is actually returning an error that is not caught.

Reply:
I think you just need to do either of these things

`err := s.db.Find(&aco, "aco_id = ?", cmsID).Error`

`	if s.db.Find(&aco, "aco_id = ?", cmsID).RecordNotFound(){
		
	}`

It is all on the [gorm docs](http://gorm.io/docs/error_handling.html)